openai/components/oai_finetune_pipeline_component.yml

```diff
@@ -0,0 +1,58 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+name: microsoft_com_azureml_oaie_finetune_pipeline_v2
+version: 0.0.1
+type: pipeline
+display_name: PipelineComponent for OpenAI Finetuning workflow
+description: Pipeline component for OpenAI finetuning workflow
+
+settings:
+  default_compute: azureml:serverless
+
+inputs:
+  model:
```

Comment:
nit: mode_id?

Reply:
no it takes model name as string input and internally resolves to model asset id

Reply:
no it takes model name as string input and internally resolves to model asset id